### DISCUSSION TITLE
如何获取“proxy.py”文件

### DISCUSSION THREAD

**Discussion by hsragi** [Melledy/Grasscutter#2445] [Q&A]
我查找了Google、百度和bing，几乎没有有关“proxy.py”的内容，所有教程也没有讲到这个玩意儿
唯一一个是在别人的仓库里，但那已经1年之久，而且也无法正常运行

**Comment by TomyJan** [comment]
🤡
https://github.com/Grasscutters/Grasscutter/blob/development/scripts/proxy.py
